### ISSUE TITLE
na_ontap_volume fails on already existing data-protection volume with no security-style set

### ISSUE THREAD

**Issue by dprts** [ansible-collections/netapp#37]
##### SUMMARY
When creating a volume with type: data-protection, the volume is created and no security style is set as expected but rerunning the task checks for existing security style and since there's nothing set it fails with:

```
raise KeyError('No element by given name %s.' % key)\nKeyError: 'No element by given name style.'\n",
```
pointing to:
```
ansible_collections/netapp/ontap/plugins/modules/na_ontap_volume.py\", line 667, in get_volume
```
This seem to affect all versions starting from 20.4.1

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
na_ontap_volume
##### ANSIBLE VERSION
```
ansible 2.9.7
```

**Comment by dprts** [comment]
The simplest fix I see is to modify 
https://github.com/ansible-collections/netapp/blob/master/ansible_collections/netapp/ontap/plugins/modules/na_ontap_volume.py#L666

```
if is_online and volume_attributes['volume-security-attributes'].get_child_by_name('style'):
```

**Comment by lonico** [comment]
We just fixed a similar issue in 20.5.0.  But it is different from this one.
We will address this one in the next release.

**Comment by lonico** [comment]
fixed with 57723e
